content/rules/asking-to-ask.md

```diff
@@ -0,0 +1 @@
+Please, don't ask to ask. Just ask your question.
```

Comment:
Might want to add more here, but the link that goes with the command really goes into the why